Sources/Widgets/Manipulators/PickerManipulator/index.js

```diff
@@ -0,0 +1,69 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkCellPicker from 'vtk.js/Sources/Rendering/Core/CellPicker';
+import vtkAbstractManipulator from 'vtk.js/Sources/Widgets/Manipulators/AbstractManipulator';
+
+// ----------------------------------------------------------------------------
+// vtkPickerManipulator methods
+// ----------------------------------------------------------------------------
+
+function vtkPickerManipulator(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkPickerManipulator');
+
+  publicAPI.handleEvent = (callData) => {
+    const { position, pokedRenderer } = callData;
+
+    model.picker.pick([position.x, position.y, 0.0], pokedRenderer);
+    if (model.picker.getActors().length > 0) {
+      model.position = model.picker.getPickedPositions()[0];
+    }
+    return {
+      worldCoords: model.position || null,
+    };
+  };
+
+  macro.setGet(publicAPI, model, ['picker']);
```

Comment:
This should be in `extend`, before `vtkPickerManipulator(publicAPI, model)`.

Reply:
Moved in [037c229](https://github.com/Kitware/vtk-js/pull/2907/commits/037c229ff87530014a624467e8c99bf26f973b67)